gamemode/modules/fadmin/fadmin/logging/sv_init.lua

```diff
@@ -11,12 +12,13 @@ FAdmin.StartHooks["Logging"] = function()
 
 		return true, OnOff
 	end)
+	logging = GetConVar("FAdmin_logging")
 end
 
 local LogFile
 function FAdmin.Log(text, preventServerLog)
 	if not text or text == "" then return end
-	if not GetConVar("FAdmin_logging"):GetBool() then return end
+	if not logging:GetBool() then return end
```

Comment:
I've not seen it happen but you could add a sanity check if you want to cover any weird and wonderful cases people seem to encounter:

```lua
if not logging then logging = GetConVar("FAdmin_logging") end
```